unleash-yggdrasil/src/strategy_parsing.rs

```diff
@@ -372,6 +372,18 @@ fn list_constraint(inverted: bool, mut node: Pairs<Rule>) -> RuleFragment {
     }
 }
 
+fn external_value(_inverted: bool, mut node: Pairs<Rule>) -> RuleFragment {
+    let strategy_index = string(node.next().unwrap());
+    Box::new(move |context| {
+        context
+            .external_results
+            .as_ref()
+            .and_then(|strategy_results| strategy_results.get(&strategy_index))
+            .copied()
+            .unwrap_or(false)
+    })
+}
```

Comment:
I'm having a hard time at how we should handle inverted but if we're consciously dropping it, we should at least state why